hubble/shell.py

```diff
@@ -172,6 +188,20 @@ def f(i):
             return "opt.%s" % i[0], str(i[1])
         # Add the args to the environment as opt.'<arg_name>'
         env.add(dict(map(f, vars(args).items())), section)
+
+        # Populate environment vars by running opt-cmd
+        # if -o was passed on the commandline
+        if args.option:
+            if 'opt-cmd' not in env:
+                log.warning("provided -o|--option, but 'opt-cmd' is not " \
+                            "defined in '%s' section" % env['section'].value)
+            else:
+                env.add(run(env['opt-cmd'].value, env))
```

Comment:
This is probably wrong, because `args.option` isn't passed to `run()`, nor was this the case in master.